app/controllers/weekly_questions_controller.rb

```diff
@@ -3,7 +3,12 @@ class WeeklyQuestionsController < ApplicationController
 
   # GET all the weekly questions with question body info
   def index
-    render json: WeeklyQuestion.all.to_json(include: {question: {only: :body}})
+    serialized = Alba.serialize(WeeklyQuestion.all) do
```

Comment:
```suggestion
    serialized = QuestionSerializer.new(WeeklyQuestion.all).serialize
```

I believe this works. No need to reimplement the whole serializer for a collection

Reply:
Yea I tried to do that way on the console and kept giving me errors, tried again on the controller but it continues giving me errors

Reply:
Which errors? I've never used Alba, to be honest, but it looks like this should work.

Reply:
Ok, I was having errors trying that way but I was doing it wrong, yesterday I tried again but I just copied your suggested change and didn't notice that it was the `QuestionSerializer` that you used instead of the `WeeklyQuestionSerializer` and that's why it gave an error, but yea now it's working that way!